modules/ztp-performance-addon-operator.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+[id="ztp-performance-addon-operator_{context}"]
+= Performance Addon Operator
+
+The Performance Addon Operator provides the ability to enable advanced node performance tunings on a set of nodes.
+
+{product-title} (OCP) provides a Performance Addon Operator to implement automatic tuning to achieve low latency
+performance for OpenShift applications. The cluster administrator uses this performance profile configuration that makes
+it easier to make these changes in a more reliable way.
+The administrator can specify whether to update the kernel to rt-kernel, the CPUs to be reserved for
```

Comment:
The list here needs to be more parallel. Suggest something like this, but up to you: The administrator can specify uptdating the kernel to rt-kernel, reserving CPUs for management workloads, and using CPUs for running the workloads.